sidecar/sidecar.go

```diff
@@ -77,6 +77,12 @@ func sidecarCommand(context *cli.Context) error {
 	if err != nil {
 		return err
 	}
+	if conf.Proxy {
+		err = conf.GenProxyConfig()
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
The logic for generating the initial NGINX config should be somewhere in the `Run` function, not at the top level here in `sidecarCommand`.